topics/statistics/tutorials/dome/tutorial.md

```diff
@@ -13,11 +13,530 @@ key_points:
+	
+	Finally, making files that detail the exact optimization protocol, including parameters and hyperparameters, publicly available is crucial. A lack of proper documentation and limited access to these records can hinder the understanding and evaluation of the model’s overall performance.[@DOME]
+
+
+## 2.1 Algorithm
+
+
+Since algorithms take input data and produce output, typically solving a particular problem or achieving a specific objective, it is essential to know which one is implemented in a study. In this way we can have better insights for the results of learning patterns, relationships, or rules that can then be applied to new, unseen data.
+Regarding ML class there are three major categories:
+
+
+- Supervised (i.e. Linear Regression, Logistic Regression, Decision Trees, Support Vector Machines (SVM) and others), 
+- Unsupervised Learning (i.e. K-Means Clustering, Principal Component Analysis (PCA) and Hierarchical Clustering and others),  
+- Reinforcement Learning (i.e. Q-Learning, Deep Q-Networks (DQN) and others). 
+
+
+**Key Questions**
+
+- __What__ is the ML algorithm class used? 
+- Is the ML algorithm __new__? 
+- If yes, why was it __chosen__ over better known alternatives?
+
+> **From Example Publication**  
+> Majority-based consensus classification based on 8 primary ML methods and post-processing.
+
+
+## 2.2 Meta-predictions
+
+
+Meta-predictions refer to predictions made by models that aggregate or utilize the outputs (predictions) of other models. Essentially, meta-prediction systems combine predictions from multiple models to produce a more robust or accurate final prediction. Meta-predictions are often used in ensemble learning techniques, where the goal is to leverage the strengths of several models to enhance overall performance.
+
+
+
+**Key Questions**
+
+- Does the model use data from __other__ ML algorithms as input? 
+- If yes, which ones? 
+- Is it clear that training data of initial predictors and meta-predictor are __independent__ of test data for the meta-predictor?
+
+> **From Example Publication**  
+> Yes, predictor output is a binary prediction computed from the consensus of other methods; Independence of training sets of other methods with test set of meta-predictor was not tested since datasets from other methods were not available.
+
+
+## 2.3 Data encoding
+
+
+Data encoding is the process of transforming data from one format or structure into another, often to make it easier for ML models or computational systems to process. 
+In ML, data often needs to be encoded to ensure that it can be effectively interpreted by algorithms, especially for algorithms that require numerical input (e.g., neural networks, SVMs).
+
+
+**Key Questions**
+
+- How were the data __encoded__ and __preprocessed__ for the ML algorithm?
+
+> **From Example Publication**  
+> Label-wise average of 8 binary predictions.
```

Comment:
It would be great if you could use GTN-style question boxes for these Key Question sections, so something like:

```suggestion
> <question-title> Key Questions </question-title>
>
> 1. How were the data __encoded__ and __preprocessed__ for the ML algorithm?
>
> > <solution-title></solution-title>
> > 1. **From Example Publication**  
> >    Label-wise average of 8 binary predictions.
> >
> {: .solution}
{: .question}
```